### DISCUSSION TITLE
FZF for zsh users

### DISCUSSION THREAD

**Discussion by mikoscz** [Selleo/commons#7] [Ideas]
https://github.com/ohmyzsh/ohmyzsh/tree/master/plugins/fzf

It's an excellent plugin that allows us fuzzy search through history in Zsh, for example, we know that we used some long command some time ago and we don't remember exactly what was the syntax, but we remember some part of it so we can open the search `C-r` type the part that we know and quickly search for the exact command.

**Comment by qbart** [comment]
i am on board with fzf, is there a way to install it universally? bash/zsh compatible without using ohmyzsh/vim plugin?

**Reply by mikoscz** [reply]
For the OMZ users its a matter of adding it to the `plugins=(fzf)` for the zinit users(me) we can add `zinit snippet OMZP::fzf`

**Reply by qbart** [reply]
so to work universally we would need some logic:
```sh
if [ using OHMYZSH ]; then
   add plugin
else
  add fzf normally
fi
```
?

then we could decide about default envs to set?

i knew that eventually we will hit the case that only works for some of people or collides with something else and we will need some conditional handling but that happened very early ;D 

let's see how others use fzf
